client/src/components/MetaDataViewerComponents/Experiments.js

```diff
@@ -26,9 +30,10 @@ const Experiments = () => {
   
   
   const orderedExperiments = [currentExperiment?.id, ...Object.keys(experimentsActivityMaps)
-    .filter(experiment => experiment !== currentExperiment?.id)];
+    .filter(experiment => experiment !== currentExperiment?.id)].filter(row =>
+    experimentAtlas[row][0] === activeAtlas.id && currentExperiment.id !== row);
```

Comment:
Issue (blocking): I would prefer if check if the activeAtlas.id is included in the experimentAtlas[row] instead of checking just the first entry